### ISSUE TITLE
[Website] Standarise the way to publish releases

### ISSUE THREAD

**Issue by Emily-Jiang** [microprofile/microprofile-marketing#225]
@Emily-Jiang commented on [Tue Aug 13 2019](https://github.com/eclipse/microprofile/issues/122)

Since spec html and pdf are not in maven central. In the past, the pdfs and htmls are attached to the releases. However, it is not convenient as they have to be downloaded for viewing. 

In the recent release architecture changes, all of the docs and javadocs are uploaded to [here](https://download.eclipse.org/microprofile/). When releasing MicroProfile reactive messaging, I used the following [format](https://github.com/eclipse/microprofile-reactive-messaging/releases/), which is much more user friendly.

I updated MicroProfile config to use this format. All other specs should adopt this format for improvement.

We also need to update microprofile.io to list all spec/pdf. I've filed [this](https://github.com/microprofile/microprofile-site/issues/87) issue on microprofile-site repo.

---

@OndroMih commented on [Tue Aug 13 2019](https://github.com/eclipse/microprofile/issues/122#issuecomment-520937374)

Ideally, the Jenkins release job would use GitHub API to generate the description for the release, including links to Javadocs and spec documents and info about the maven artifact.

**Comment by Emily-Jiang** [comment]
@rstjames can we meet up to discuss the layout?

**Comment by rstjames** [comment]
@Emily-Jiang When would you all like to meet?

**Comment by rstjames** [comment]
@Emily-Jiang Emily and I met,
She is looking for a page that would be on MP site that would list all specs with content bellow. (big spec overview)
Example: (content filler)
<img width="990" alt="Screen Shot 2019-11-19 at 11 37 25 AM" src="https://user-images.githubusercontent.com/12132146/69139538-03fb7b00-0ac1-11ea-9de3-665b25c17583.png">

We will continue this conversation on the next marketing call and community call. Then we can get others thoughts on this topic and decide what the next steps could be.

**Comment by Emily-Jiang** [comment]
I am thinking to have two layers:
First layer contains all MP Spec, basically the building blocks(Config, Fault Tolerance, Rest Client, etc). On clicking each individual spec, it lists all versions with the links to javadocs, html, pdf.

**Comment by Emily-Jiang** [comment]
I will like to make some progress with this. @rstjames what is the next step?

**Comment by Emily-Jiang** [comment]
@rstjames what's the next stage?

**Comment by rstjames** [comment]
@Emily-Jiang that is a good question. I am not sure... I can not give away resources (if we have any to spare) without discussing things with @aeiras . This may take a couple weeks because of holidays.

**Comment by aeiras** [comment]
Hey you two @Emily-Jiang and @rstjames ,

I recommend a deep discussion on next MP Marketing hangout to see exactly what it will take to proceed on this task. Without knowing specifics, we cannot provide Ryan's time to work on a blind task. 

Recommendation is always to write up the plan via drive website document, take on priorities and drop anything shiny or "not vital".  Once that document is ready, bring it to the Community call for consumption and that will trigger action. 

Cheers,

**Comment by aeiras** [comment]
I added Cesar and myself to this ticket :) for reference!

**Comment by aeiras** [comment]
new ticket takes into account this feedback with reference to 3.3 written retrospective! https://github.com/eclipse/microprofile-marketing/issues/251

The project page under the website needs to be 100% automatic and link to all the active repos. 
https://microprofile.io/projects/

Can someone in this ticket find out whats needed? :)

**Comment by aeiras** [comment]
@rstjames, 

Can you please make a call to action bottom on the Project's page that brings the viwer to the MP main Repo: https://github.com/eclipse?utf8=✓&q=microprofile

Thanks

**Comment by rstjames** [comment]
@aeiras button added to the https://microprofile.io/projects/.

**Comment by aeiras** [comment]
thanks Ryan. Small improvement for the time being.

**Comment by aeiras** [comment]
We tackled this today during Mktg hangout. Tackled with new ticket. 
The mktg side has its wiki & the technical side is being updated to know how to communicate with branding! https://github.com/eclipse/microprofile-marketing/issues/277

**Comment by Emily-Jiang** [comment]
@rstjames can you create a draft for this web page? Please let me know what info you need from me. Can we set up a deadline? How long do you think this will take @rstjames ?

**Comment by aeiras** [comment]
Hey @Emily-Jiang, 

We have made improvements on this area. Do you believe we still need this ticket or can we close it and iterate via new issues to tackle improvements on the work done?

**Comment by aeiras** [comment]
@Emily-Jiang, 

Now that your formally took a part of the pushing on release MP. 4.0 which includes marketing templates, do we still need this ticket or can we focus the work on a new ticket and go by layers?

I recommend we choose layers and allow for smaller releases to become the norm as well.

**Comment by aeiras** [comment]
Emily, 

we have evolved this and the latest work deals with this ticket under the MPWG. 
I will close it.
